api.py

```diff
@@ -47,9 +48,11 @@ class TimecodeFramerate(enum.Enum):
 
 
 class MoveshelfApi(object):
-    def __init__(self, api_key_file='mvshlf-api-key.json', api_url = 'https://api.moveshelf.com/graphql'):
+    def __init__(self, api_key_file, api_url = 'https://api.moveshelf.com/graphql'):
```

Comment:
@roksanagajek why is this changed here?

Reply:
because in your code the key had a predefined name (mcshlf-api-key.json), in my code you can give the key the name you want since you select the key

Reply:
the part you deleted should only be used if no value is provided. It should work anyway if you provide a specific value (i.e. calling `MoveshelfApi('/mypath/mykey.json')` will in both cases give `api_key_file` the value `'/mypath/mykey.json'`